about.html

```diff
@@ -15,6 +15,6 @@
 
 <p>You can find further information on our instance in our <a href="https://github.com/chaossocial/about">GitHub repository</a>, which also provides a <a href="https://github.com/chaossocial/about/blob/master/blocked_instances.md">list</a> of currently blocked or silenced instances, alongside with reasons.</p>
 
-<p>We (@leah and @rixx) are maintaining this instance on our spare time, hardware and nerves. Don't push either of those.</p>
+<p>We (<a href="https://chaos.social/@leah">@leah</a> and <a href="https://chaos.social/@rixx">@rixx</a>) are maintaining this instance on our spare time, hardware and nerves. Don't push either of those.</p>
 
-<p>This site is currently running on mastodon <em>v1.2.2</em></p>
+<p>This site is currently running on mastodon <em>v1.4.3</em></p>
```

Comment:
I'd argue that this is not necessary – the version number is provided in the sidebar right next to the "about" text, and in the text we'd need to remember to manually update it every time.